go.sum

```diff
@@ -10,6 +10,8 @@ github.com/armon/go-radix v1.0.0 h1:F4z6KzEeeQIMeLFa97iZU6vupzoecKdU5TX24SNppXI=
 github.com/armon/go-radix v1.0.0/go.mod h1:ufUuZ+zHj4x4TnLV4JWEpy2hxWSpsRywHrMgIH9cCH8=
 github.com/aws/aws-sdk-go v1.17.0 h1:+pbWEdKxH1qlLb07as1+auEVvx+IxkaDzQLwMzbK1tI=
```

Comment:
Hmm. When I ran `go mod tidy` the v1.17.0 were removed. We'll likely be setting up renovatebot to handle this going forward so hopefully the Go Modules quirks will be reduced. 🤞